cmd/snap/cmd_prepare_image.go

```diff
@@ -49,8 +50,17 @@ func init() {
 }
 
 func (x *cmdPrepareImage) Execute(args []string) error {
+	var modelFile, modelRef string
+
+	if osutil.FileExists(x.Positional.ModelAssertion) {
```

Comment:
I learned something here. Apparently the comment is ordered according to the moment it was saved, rather than the order in the code. The suggestion is actually above.